personio-personnel-data-api-oa3.yaml

```
@@ -1123,7 +1123,7 @@ paths:
         - name: offset
           in: query
           required: false
-          description: Pagination attribute to identify which page you are requesting, by the form of telling an offset from the first record that would be returned.
+          description: The offset from the first record that would be returned. With 3 results and an offset of 1, 2 results are returned.
```

maybe split `1, 2` in the sentence
```suggestion
          description: The offset from the first record that would be returned. With total of 3 results and provided offset of 1, two results will be returned.
```